src/ParaTest/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -321,15 +338,14 @@ protected function getDefects(array $defects, $type)
     /**
      * Prints progress for large test collections
      */
-    protected function printProgress()
+    protected function getProgress()
     {
-        printf(
-            ' %' . $this->numTestsWidth . 'd (%3s%%)',
+        return sprintf(
+            ' %' . $this->numTestsWidth . 'd / %' . $this->numTestsWidth . 'd (%3s%%)',
             $this->casesProcessed,
-            floor(($this->casesProcessed / $this->totalCases) * 100)
+            $this->totalCases,
```

Comment:
I can remove code related to total count correction and write also S in progress for incomplete/skipped tests. But it is related to bug with wrong rendering of incomplete/skipped tests  =)